tools/src/ProjectVersions.ts

```diff
@@ -42,7 +42,7 @@ export async function androidAppVersionAsync(): Promise<string> {
 }
 
 export async function getHomeSDKVersionAsync(): Promise<string> {
-  const homeAppJsonPath = path.join(EXPO_DIR, 'home', 'app.json');
+  const homeAppJsonPath = path.join(EXPO_DIR, 'apps', 'expo-go', 'app.json');
```

Comment:
Since this path is now used in a few places, maybe we can export it as `EXPO_GO_DIR` from `Constants.ts`?

Reply:
Will do in a follow-up!

Reply:
https://github.com/expo/expo/pull/26280